README.md

```diff
@@ -61,7 +61,7 @@ After the data has been inserted, you can finally start to query the database.
 
 ```js
 const searchResult = search(db, {
-  term: 'if',
+  term: 'If',
```

Comment:
Hi @AlessandroMinoccheri 👋🏽 
I've looked into it (as I'm writing the docs for this). 
It seems Lyra starts being case insensitive by default only when the `term`.length > 2.

```js
const db = create({
  schema: {
    title: 'string',
  }
});

insert(db, {
  title: 'What If',
});

insert(db, {
  title: 'what if',
});

const searchResult = search(db, {
  term: 'if',
  properties: '*'
});
```
Result

```sh
{
  elapsed: 87334n,
  hits: [ { id: '23490458-8', word: 'what if' } ],
  count: 1
}
```

When the term.length > 2

```js
const db = create({
  schema: {
    word: 'string',
  }
});

insert(db, {
  word: 'air',
});

insert(db, {
  word: 'Air',
});

const searchResult = search(db, {
  term: 'air',
  properties: '*'
});
```
result

```bash
{
  elapsed: 89416n,
  hits: [
    { id: '23309215-1', word: 'air' },
    { id: '23309215-5', word: 'Air' }
  ],
  count: 2
}
```

Interesting.

Reply:
@thomscoder you are right! It could be interesting to fix this problem. I am trying to understand the logic of the search.

Reply:
I opened an issue for this problem in the meanwhile

Reply:
> I opened an issue for this problem in the meanwhile

Perfect, thank you ! 😄

Reply:
@thomscoder I tried to fix it but I don't know if is the right way, if not I would like to discover together the right way.
https://github.com/nearform/lyra/pull/59